tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+          type: string
+          required: true
+      responses:
+        204:
+          description: No Content
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Selection with the given ID not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/update-tags:
+    post:
+      tags:
+        - tags
+      summary: 'Create or update multiple tags'
+      description: Create or update multiple tags in one request.
+      operationId: CreateOrUpdateTags
+      parameters:
+        - in: body
+          name: tagsAndMerge
+          description: The tags to create or update, and a flag specifying whether keywords and properties should be merged for updates.
+          schema:
+            type: object
+            title: Tag List and Merge Flag
+            properties:
+              tags:
+                description: List of tags
+                type: array
+                items:
+                  $ref: '#/definitions/Tag'
+              merge:
+                description: 'If set, performs a merge of any specified Keywords or Properties.'
+                type: boolean
+          required: true
+      responses:
+        201:
+          description: Created
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: 'Conflict: Type change not supported'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags:
+    post:
+      tags:
+        - tags
+      summary: Create or update a tag
+      description: Create or update single tag. Performs a merge of any specified Keywords or Properties. Changing a tag's type is not supported.
+      operationId: CreateTag
+      parameters:
+        - in: body
+          name: tags
+          description: Tag to create
+          schema:
+            $ref: '#/definitions/Tag'
+          required: true
+      responses:
+        201:
+          description: 'Created: Tag created successfully'
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: 'Conflict: Type change not supported'
+        default:
+          $ref: '#/responses/Error'
+    get:
+      tags:
+        - tags
+      summary: Get multiple tags
+      description: Get a list of tags, filtered based on the provided query parameters.
+      operationId: GetTags
+      parameters:
+        - in: query
+          name: path
+          description: Comma-separated list of tag paths. May include glob-style wildcards
+          type: string
+          required: false
+        - in: query
+          name: keywords
+          description: Comma-separated list of case-sensitive keywords.
+          type: string
+          required: false
+        - in: query
+          name: properties
+          description: Comma-separated list of name=value strings, e.g. key1=value,key2=value.
```

Comment:
Do these lists have spaces after the commas or no? Also, idk why this line is yellow .